cms/static_src/sass/main.scss

```diff
@@ -2,3 +2,18 @@
 body {
     display: block;
 }
+
+// Custom CSS from Wagtail doc to enable responsive video embeds
+.responsive-object {
+    position: relative;
+}
+
+.responsive-object iframe,
+.responsive-object object,
+.responsive-object embed {
+    position: absolute;
+    top: 0;
+    left: 0;
```

Comment:
```suggestion
    left: 0;  // styleling-disable-line property-disallowed-list
```

should sort this

Reply:
Yes, I found this fix yesterday and it fixes the front-end linting issue. I was going to ask about the best practice moving forward, so I didn't push this up.

But now I see that we are using the rules from `stylelint-config-wagtail` and it has that `property-disallowed-list-rule`.

Thanks. Will push up the changes.

Reply:
Done!